e2e/testenv/infra/cmd/setup.go

```diff
@@ -268,12 +269,20 @@ func spinUpOCIRegistry(k kubectl.Command, wg *sync.WaitGroup) {
 		fail(fmt.Errorf("spin up OCI registry: %v", err))
 	}
 
-	out, err := k.Apply("-f", testenv.AssetPath("helm/zot_secret.yaml"))
+	var err error
+	htpasswd := "fleet-ci:$2y$05$0WcEGGqsUKcyPhBFU7l07uJ3ND121p/FQCY90Q.dcsZjTkr.b45Lm"
+	if os.Getenv("CI_OCI_USERNAME") != "" && os.Getenv("CI_OCI_PASSWORD") != "" {
+		if p, err := bcrypt.GenerateFromPassword([]byte(os.Getenv("CI_OCI_PASSWORD")), bcrypt.MinCost); err == nil {
+			htpasswd = fmt.Sprintf("%s:%s", os.Getenv("CI_OCI_USERNAME"), string(p))
+		}
```

Comment:
Should we output an error here in case one happens? Not sure how likely that outcome is, but since we're relying on user input...